docs/language/accounts.md

```diff
@@ -110,7 +110,7 @@ Account storage is accessed through the following functions of `AuthAccount`.
 This means that any code that has access to the authorized account has access
 to all its stored objects.
 
-- `cadence•fun save<T>(_ value: T, to: Path)`
```

Comment:
We've recently migrated all the documentation, and one of the features I planned to have there was support for inline code blocks with syntax highlighting support.

I've just added it this morning in https://github.com/onflow/flow/pull/48, so the language tag here and in the other cases below should not be removed.

Reply:
Sure, it occurred to me that this could be the case. I'll simply remove the commit.